src/WebrootPreloadMiddleware.php

```diff
@@ -96,7 +96,9 @@ public function __invoke(ServerRequestInterface $request, callable $next)
             if ($request->hasHeader('If-Match')) {
                 $expectedEtag = current($request->getHeader('If-Match'));
                 $expectedEtag = trim($expectedEtag, '"');
-                if ($expectedEtag !== $item['etag']) {
+                foreach($expectedEtag as $item)
+                {
+                $expectedEtag !== $item['etag'];
                     return new Response(412);
                 }
             }
```

Comment:
Not entirely what I meant, meant something along the lines of 

```suggestion
if ($request->hasHeader('If-Match')) {
    foreach ($request->getHeader('If-Match') as $expectedEtag ) {
        $expectedEtag = trim($expectedEtag, '"');
        if ($expectedEtag !== $item['etag']) {
             return new Response(412);
        }
    }
}
```

Reply:
Oh OK.